src/handlers/regional.py

```diff
@@ -26,10 +27,14 @@ def get(self):
     regions_missing_championships = [
         region for region in regions if region.key not in championship_regions]
 
+    #If not one of the regionals has happened yet, then show the previous year's champions
+    champion_year = year-1 if len(regions_missing_championships) == 7 else year 
```

Comment:
I think this is actually "if not one of the regionals has been announced yet".

How about looking at championships[0].start_date instead?

Reply:
Ah yes, I see what you mean. I can fix this, but to test it properly, I need to play around with #204 for a bit longer.